cogment_verse/web/web_app/src/hooks/useRealTimeUpdate_v2.jsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 AI Redefined Inc. <dev+cogment@ai-r.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useEffect, useState } from "react";
+
+export const useRealTimeUpdateV2 = (fps = 30, paused = true) => {
```

Comment:
what's the difference with `useRealTImeUpdate` ?

Reply:
it does not take `sendAction` as an argument. I finally use `useRealTimeUpdate` and remove `useRealTimeUpdate_v2`